smartsim/ml/data.py

```diff
@@ -406,72 +369,15 @@ def init_samples(self, init_trials: int = -1, wait_interval: float = 10.0) -> No
         if self.shuffle:
             np.random.shuffle(self.indices)
 
-    def _data_exists(self, batch_name: str, target_name: str) -> bool:
-        if self.need_targets:
-            return all(
-                self.client.tensor_exists(datum) for datum in [batch_name, target_name]
-            )
-
-        return bool(self.client.tensor_exists(batch_name))
+    def _data_exists(self, batch_name: str, target_name: str) -> None:
+        pass
```

Comment:
Same question as before -- for these functions where we are deleting function content but not the function, should we raise some kind of error about this needing to be inmplemented for SmartSimV1.0?  Maybe a custom error class so we can easily search for all of the instances?

Reply:
Per group discussion, we are not doing this.  We will rely on ``pass``